src/main/java/org/galatea/starter/service/PriceService.java

```diff
@@ -20,13 +42,101 @@
    * @return the result of executing the command with the given parameters
    */
 
-  public String processStock(String stock, Integer daysToLookBack) {
 
-    String parameters = stock + " " + daysToLookBack;
-    return parameters;
+  @Autowired
+  private PricesClient pricesclient;
+  public long processTime;
```

Comment:
remove everything that isn't the PricesClient. see if you can make this work by passing around parameters.